### DISCUSSION TITLE
Qiitaの目次はなぜ小さいの？（解決済み）

### DISCUSSION THREAD

**Discussion by ktyaman** [increments/qiita-discussions#238] [要望・アイデア・不具合]
### 知りたいこと
Qiitaで長い記事を読んでいると、いつも次のようなことを思わずにいられません。
「**画面右下の目次、小さいなあ**」
「**上に余白があるから、上まで目次のスペースを広げてほしいなあ**」
「**何か理由があるのかな？**」

この理由を教えていただけないでしょうか。

### 疑問の発端
一例として次の画像をご覧ください
![exmpl](https://user-images.githubusercontent.com/93898059/173559429-db553c7f-ffa4-419c-ab19-981468db5be3.png)

画像の右下に目次がありますが、すべての見出しが見えていませんよね。

**チートシートなど見出しがたくさんある記事では、目的のセクションを見つけるために目次をスクロールする必要があります。**

**マウスホイールを少し回しただけで結構 (4行) 動くので、探す作業がやりづらいと感じています。**

（マウスホイールのスクロールの設定によりますが、私の場合、Acrobat Readerなど他の場面ではちょうどよい設定になっていて、そちらを優先しているわけです）。

### 提案
目次のスペースを今より大きくしてみてほしいです。

あるいは、可能なら、スクロール幅を変えることも一つの案になりえます。

**Comment by munierujp** [comment]
>「上に余白があるから、上まで目次のスペースを広げてほしいなあ」

そこの余白は広告表示エリアだと思います。

<img width="1250" alt="スクリーンショット 2022-06-14 20 29 51" src="https://user-images.githubusercontent.com/20086673/173567539-cabd4ea4-9cbc-424c-9804-b9d96f46a5a0.png">

**Reply by ktyaman** [reply]
おや、その可能性を考えていくつかの記事を見たのですが、まったく表示されませんでした。

その理由が予想できれば教えていただけませんか？

特に広告ブロックに関する何かを施した記憶はないのですが...

**Reply by munierujp** [reply]
一昔前はログインしていれば表示されなかった気がするのでもしかして……と思いましたが、こちらの環境だとログイン状態でも表示されますね 🤔

**Reply by ktyaman** [reply]
なるほど。
重ねた質問にもご対応いただき、ありがとうございます！

**Reply by tanaka-asami415** [reply]
@ktyaman 
お問い合わせありがとうございます🙏

munierujp さんのおっしゃる通り、広告枠となっております🙇

一部ブラウザにはデフォルトで広告をブロックするものもあるようですので
一度ブラウザの設定をご確認いただけないでしょうか🙏

ブラウザの設定をされていない場合は、不具合でないか調査をしたいため、
ご使用のブラウザとバージョンを教えていただけると幸いです🙏

**Reply by ktyaman** [reply]
なるほど、ブラウザに元から付いていることがあるのですね。

結果から言えば、ウイルスバスターの拡張機能 (Trend ツールバー) によって自動的にブロックされていました。

ちなみに、ウイルスバスターを停止して Google Chrome (バージョン 102.0.5005.115) の広告ブロック機能だけにした場合は、広告が表示されました。

ご対応ありがとうございます。

**Reply by tanaka-asami415** [reply]
@ktyaman 
ご返信ありがとうございます！原因がわかって安心いたしました:blush:
本件、クローズさせていただきます🙏
